datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/manifest.py

```diff
@@ -50,6 +50,10 @@ def get_manifest_schema():
                     "description": "The metric to use to determine the health of this integration",
                     "oneOf": [{"type": "string"}, {"type": "array", "items": {"type": "string"}}],
                 },
+                "metric_integration_check_name": {
+                    "description": "The name of an external integration check providing metrics",
+                    "type": "string",
+                },
```

Comment:
With the latest changes we don't need to define this anymore right?

Reply:
https://user-images.githubusercontent.com/45568537/102638225-740e8180-4157-11eb-91d6-97e7ef8e23ca.mp4